### DISCUSSION TITLE
PROCESS-NAME规则部分自定义端口不生效

### DISCUSSION THREAD

**Discussion by sino1641** [MetaCubeX/mihomo#1695] [Q&A] [answered]
大佬们，
因为运营商问题，需要teams过墙，
最新的mihomo 1.18.10，打开tun模式，配置片段如下：

```yaml
  - PROCESS-NAME,ms-teams.exe,Microsoft
  - MATCH,Proxies
```
其中Microsoft和Proxies配置到两个不同的节点。

重启核心后测试，发现以下的情况：
443端口可以正常命中进程名规则，但是自定义端口还是会导向MATCH
请问是by design还是需要额外的规则控制？感谢！

<img width="804" alt="Snipaste_2024-12-04_16-47-19" src="https://github.com/user-attachments/assets/d3cef1f7-cce2-4a24-b7ad-3a46274d73e0">

**Comment by xiandu999-yu** [comment, marked as answer]
- AND,((PROCESS-NAME,ms-teams.exe),(DST-PORT,1-65535),(SRC-PORT,1-65535)),Microsoft
试试？
